### PR TITLE
Updated job cards

### PR DESCRIPTION
**Pull request by devdog72**

### CODE REVIEW

File: lib/public/main.css [hunk 1]
```diff
@@ -296,4 +302,68 @@ textarea {
   border: 1px solid #ff7979;
   box-shadow: inset 0 0 5px #ff7979;
   outline: none;
-}
\ No newline at end of file
+}
+
+/* Media query modifications when display < 500px */
+@media screen and (max-width: 500px) {
+
```
Comment (thesimpledev) [inline lib/public/main.css]: Remove line

File: lib/public/main.css [hunk 2]
```diff
@@ -296,4 +302,68 @@ textarea {
...
+  .job-card {
+    position: relative;
+    display: block;
+  }
+
+  .job-location {
+    position: absolute;
+    right: 10px;
+    top: 6px;
```
Comment (thesimpledev) [inline lib/public/main.css]: Top and right should be the same

File: lib/public/main.css [hunk 3]
```diff
@@ -296,4 +302,68 @@ textarea {
...
+  }
+
+  .keyword {
+    padding: 3.0px;
```
Comment (thesimpledev) [inline lib/public/main.css]: `3` instead of `3.0`

File: lib/public/main.css [hunk 4]
```diff
@@ -296,4 +302,68 @@ textarea {
...
+  }
+
+  .job-position {
+    margin-bottom: 3.0px;
```
Comment (thesimpledev) [inline lib/public/main.css]: `3` instead of `3.0`

File: lib/public/main.css [hunk 5]
```diff
@@ -296,4 +302,68 @@ textarea {
...
+    align-items: flex-start;
+  }
+
+  .nav-logo span {
+    margin-left: 0px;
+  }
+
+  .job-points-passing {
+    padding: 3px;
+
```
Comment (thesimpledev) [inline lib/public/main.css]: remove line

File: lib/public/main.css [hunk 6]
```diff
@@ -296,4 +302,68 @@ textarea {
...
+  }
+
+  .job-link {
```
Comment (thesimpledev) [inline lib/public/main.css]: remove selector

File: lib/public/main.css [hunk 7]
```diff
@@ -296,4 +302,68 @@ textarea {
...
+
+  }
+
```
Comment (thesimpledev) [inline lib/public/main.css]: remove line

File: lib/public/main.css (hunk 1 above)
Comment (devdog72) [reply lib/public/main.css]: _Line removed_

File: lib/public/main.css (hunk 2 above)
Comment (devdog72) [reply lib/public/main.css]: Both coordinates set to 6px.

File: lib/public/main.css (hunk 3 above)
Comment (devdog72) [reply lib/public/main.css]: Modified

File: lib/public/main.css (hunk 4 above)
Comment (devdog72) [reply lib/public/main.css]: Modified

File: lib/public/main.css (hunk 5 above)
Comment (devdog72) [reply lib/public/main.css]: Line removed

File: lib/public/main.css (hunk 6 above)
Comment (devdog72) [reply lib/public/main.css]: Empty Selector removed from code

File: lib/public/main.css (hunk 7 above)
Comment (devdog72) [reply lib/public/main.css]: Line removed

File: lib/public/main.css [hunk 8]
```diff
@@ -296,4 +302,61 @@ textarea {
...
+    position: absolute;
+    right: 6px;
+    top: 6px;
+  }
+
+  .keyword {
+    padding: 3px;
+  }
+
+  .job-position {
```
Comment (thesimpledev) [inline lib/public/main.css]: Merge these selectors

**Pull request merged**